sw/device/silicon_creator/lib/epmp_unittest.cc

```diff
@@ -2,7 +2,7 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 
-#include "sw/device/silicon_creator/lib/epmp.h"
+#include "sw/device/silicon_creator/lib/epmp_state.h"
```

Comment:
Hmm, I think the call-sites in this file should also be updated.

Reply:
Next commit